chef/automate
# Moving Org and Chef Server filtering to the search bar

### 📋 TODO
- [x] notification update to URL
- [x] Add route change. 
### :nut_and_bolt: Description: What code changed, and why?
To update the Client Runs left sidebar to not have the chef servers and organization selector. Add the chef servers and organization filters to the search bar. 

### :chains: Related Resources
#692 

### :+1: Definition of Done
* The left sidebar is updated to look like https://user-images.githubusercontent.com/5489125/60611799-0a8c8e80-9d7c-11e9-8dcb-6004cf9a6a04.png with the chef servers and organization selector remove. The chef servers and organization filters are in the Client Runs search bar. 
* Update the URL route to the client runs page to `/infrastructure/client-runs`. 
* Updated the notifications messages to point to the updated client runs URL 

### :athletic_shoe: How to Build and Test the Change
1. `build components/config-mgmt-service && build components/automate-gateway build components/automate-ui && start_all_services`
1. `chef_load_nodes 100`
1. https://a2-dev.test/infrastructure/client-runs Ensure the left sidebar look correct
1. In the search bar ensure the Organization and Chef Server filter correctly
1. create a notification and send it to the requestbin http://requestbin.sjcmmsn.com/
1. Send a failed request with `send_chef_run_failure_example`.
1. Ensure the message field "automate_failure_url has the correct URL to the client runs page. 

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
![Screen Shot 2019-07-29 at 4 19 26 PM](https://user-images.githubusercontent.com/1679247/62089167-cbd3e200-b21c-11e9-9c9c-039235626bc8.png)

![Screen Shot 2019-07-30 at 11 00 51 AM](https://user-images.githubusercontent.com/1679247/62153407-510cd480-b2b9-11e9-9701-f0bdd744e6f2.png)

![Screen Shot 2019-08-06 at 9 16 01 AM](https://user-images.githubusercontent.com/1679247/62557079-224bac80-b82b-11e9-837a-ed2abd31c79b.png)
